### ISSUE TITLE
COLOR PICKER - Keyboard user is unable to access the “Font color“ Sliders

### ISSUE THREAD

**Issue by sagaranandani17** [OfficeDev/office-ui-fabric-react#11060]
### Environment Information

- **Package version(s)**: 7.24.0
- **OS version:** 1903 (OS Build: 18326.295)
- **Browser:** Edge(Microsoft Edge 44.18362.267.0)

**User Experience:** 
If controls are not accessible with keyboard than keyboard user will not be able to access the control and will face difficulties while interacting with it.

**Actual behavior:**
Keyboard user is unable to access the “Font color “ Sliders in color picker

**Expected behavior:**
Keyboard user should able to access the “Font color “ Sliders and change color using keyboard.

![image](https://user-images.githubusercontent.com/57341535/68118933-4e102880-ff27-11e9-8939-c0bf42a16e3f.png)

**Comment by ecraig12345** [comment]
This was also fixed by #10882.

**Comment by ecraig12345** [comment]
Also, this issue is a duplicate of #8012. Please be sure to search for existing issues before opening a new one.
